targets/generic

```diff
@@ -59,6 +59,8 @@ config('KERNEL_SECCOMP', false)
 -- use try_config, so enabling the package is still possible
 try_config('PACKAGE_kmod-mt7915e', false)
 
+config('OONF_GENERIC_HTTP', true)
```

Comment:
this needs to be enabled when mesh olsrd is used so http module in olsr2 gets enabled (used for nodeinfo, which I'll pr later). is there any way I can enable it via the package? it breaks gluon build otherwise

Reply:
perhaps try_config?